conf/nebula-graphd.conf.production

```diff
@@ -53,4 +53,12 @@
 # HTTP2 service port
 --ws_h2_port=13002
 
---load_data_interval_secs=120
+--heartbeat_interval_secs=10
```

Comment:
What's this option for?

Reply:
This is the time interval of communication between meta client and graphd service

Reply:
@dutor load_data_interval_secs has been removed.